docs/advanced_topics/documents/custom_document_model.md

```diff
@@ -0,0 +1,68 @@
+(custom_document_model)=
+
+# [Custom document model](custom_document_model)
+
+An alternate `Document` model can be used to add custom behaviour and
+additional fields.
+
+You need to complete the following steps in your project to do this:
+
+- Create a new document model that inherits from
+  `wagtail.documents.models.AbstractDocument`. This is where you would
+  add additional fields.
```

Comment:
not sure if these line breaks will break the list item, maybe bring these two lines here into the above line.